--- a/pacman.conf
+++ b/pacman.conf
@@ -75,14 +75,14 @@ LocalFileSigLevel = Optional
 #[core-testing]
 #Include = /etc/pacman.d/mirrorlist
 
-[core]
-Include = /etc/pacman.d/mirrorlist
+# [core]
+# Include = /etc/pacman.d/mirrorlist
 
 #[extra-testing]
 #Include = /etc/pacman.d/mirrorlist
 
-[extra]
-Include = /etc/pacman.d/mirrorlist
+# [extra]
+# Include = /etc/pacman.d/mirrorlist
 
 # If you want to run 32 bit applications on your x86_64 system,
 # enable the multilib repositories as required here.
@@ -98,3 +98,7 @@ Include = /etc/pacman.d/mirrorlist
 #[custom]
 #SigLevel = Optional TrustAll
 #Server = file:///home/custompkgs
+
+[offline]
+SigLevel = Optional TrustAll
+Server = file:///var/cache/omarchy/mirror/offline/
