--- a/profiledef.sh
+++ b/profiledef.sh
@@ -21,7 +21,10 @@ file_permissions=(
   ["/root"]="0:0:750"
   ["/root/.automated_script.sh"]="0:0:755"
   ["/root/.gnupg"]="0:0:700"
+  ["/root/configurator"]="0:0:755"
+  ["/var/cache/omarchy/mirror/offline/"]="0:0:775"
+  ["/root/check_connectivity.sh"]="0:0:755"
   ["/usr/local/bin/choose-mirror"]="0:0:755"
-  ["/usr/local/bin/Installation_guide"]="0:0:755"
-  ["/usr/local/bin/livecd-sound"]="0:0:755"
+  ["/usr/bin/tte"]="0:0:755"
+  ["/usr/lib/omarchy-installer"]="0:0:755"
 )
